cvat/apps/engine/views.py

```diff
@@ -559,6 +563,66 @@ def dump(self, request, pk, filename):
 
         return Response(status=status.HTTP_202_ACCEPTED)
 
+    @action(detail=AttributeError, methods=['POST'], url_path='request-extra-annotation',
+            serializer_class=RequestExtraAnnotationSerializer)
+    def request_extra_anno(self, request, pk):
+        db_task = self.get_object()
+        params_serializer = RequestExtraAnnotationSerializer(data=request.data, context={"task": db_task})
+        params_serializer.is_valid(raise_exception=True)
+        segments = params_serializer.validated_data["segments"]
+        assignees = params_serializer.validated_data["assignees"]
+
+        try:
+            request_extra_annotation(db_task, segments, assignees)
+        except FailedAssignmentError as e:
+            failed_segments = [s.id for s in e.failed_segments]
+            data = {"message": "Can't find assignees for the given segments", "segments": failed_segments}
+            return Response(data=data, status=status.HTTP_406_NOT_ACCEPTABLE)
+
+        return Response(status=status.HTTP_202_ACCEPTED)
+
+    @action(detail=True, methods=['POST'], url_path='merge-annotations')
+    def merge(self, request, pk):
```

Comment:
you import `from .ddln.multiannotation import request_extra_annotation, FailedAssignmentError, merge`
use different name to disambiguate or import the containing package via alias